mock.go

```diff
@@ -219,7 +219,7 @@ type mockChainIO struct {
 var _ lnwallet.BlockChainIO = (*mockChainIO)(nil)
 
 func (m *mockChainIO) GetBestBlock() (*chainhash.Hash, int32, error) {
-	return activeNetParams.GenesisHash, m.bestHeight, nil
+	return nil, m.bestHeight, nil
```

Comment:
Should we just return `testnetParams.GenesisHash` here to not change the behavior? Not sure it's even used now, though.

Reply:
I feel like this is fine because no tests failed in removing this

Reply:
Could keep the same behavior but then we need to have `mockChainIO` store a `bitcoinNetParams` struct, or at least the genesis hash

Reply:
I don't think the default value of `activeNetParams` was overwritten in any of the tests that use the mock. So we really just could hard code the testnet value here to simplify things but keep the old behavior.

Reply:
Hardcoded testnet value